### ISSUE TITLE
Separate small cap fonts?

### ISSUE THREAD

**Issue by jrmiller82** [adobe-fonts/source-serif-pro#33]
Any plans on adding true small caps to the font faces?

**Comment by frankrolf** [comment]
“True” Small Caps already a are feature of Roman weights.
Small Caps are not very important in Italics IMO, but I will use this issue to gauge interest.

FYI: You made the same request before (#24), and the same answer was given.

**Comment by jrmiller82** [comment]
My apologies; I guess, my question is how to use them in Libreoffice of Word then?

**Comment by frankrolf** [comment]
That I don’t know. I found the following:
https://ask.libreoffice.org/en/question/97858/small-caps-how-to-do-it/

**Comment by jrmiller82** [comment]
Let me rephrase, can we get the small caps separated out as separate fonts instead of combined in the existing ones?

**Comment by moyogo** [comment]
@jrmiller82 
One can use [pyftfeatfreeze](https://github.com/twardoch/fonttools-utils/tree/master/pyftfeatfreeze) to generate a font with the small caps glyphs as the default by “freezing” the OpenType small cap feature.

**Comment by PeterBocan** [comment]
@miguelsousa so, what's the focus now?

**Comment by miguelsousa** [comment]
@PeterBocan read [Frank's blog post](https://blog.typekit.com/2018/08/16/source-serif-italics/). There's a lot of other stuff he's working on.

**Comment by samboy** [comment]
For people using word processors and what not which do not easily support OpenType small caps, I have made a TrueType version of a subset of Source Serif Pro Roman where lower case letters are small caps: https://github.com/samboy/NotAtAllOfficialFont 

Only English and Spanish are supported in this font, and it’s missing things like the Euro symbol (this was a quick and dirty conversion)

**Comment by samboy** [comment]
I have added small caps bold to https://github.com/samboy/NotAtAllOfficialFont

**Comment by kenmcd** [comment]
LibreOffice has supported OpenType features since v5.3.   
To display small caps you simply add ":smcp" to the font name when applied.
So it looks like this: Source Serif Pro:smcp
Just tested this and it is working fine with this font.
See attached screenshot.
![source serif pro - small caps test](https://user-images.githubusercontent.com/844920/44364423-d0741000-a47b-11e8-81fa-4a75bb15317d.jpg)

**Comment by jrmiller82** [comment]
Those are the true small caps? And those aren't just ticking the "small caps" box generated font from the word processor? https://practicaltypography.com/small-caps.html

**Comment by kenmcd** [comment]
Yes. This only works if the font has real small caps characters available and the OpenType feature is enabled in the font file.
Both of these are true with this font. I checked it.
This is not like the fake small caps button.

**Comment by samboy** [comment]
My own testing, using the SCTest font I just added to https://github.com/samboy/NotAtAllOfficialFont shows that:

* Microsoft Word uses fake small caps with this font
* SCTest:smcp is using real small caps in Libre office.

**Comment by jrmiller82** [comment]
Thank you.

**Comment by kenmcd** [comment]
Note that if you add ":smcp" to a font that does not have the OpenType small caps feature available - nothing happens.
There is no faking possible with this method.

**Comment by jigillespie** [comment]
> One can use [pyftfeatfreeze](https://github.com/twardoch/fonttools-utils/tree/master/pyftfeatfreeze) to generate a font with the small caps glyphs as the default by “freezing” the OpenType small cap feature.

Thank you for the suggestion! It's a fantastic tool for making a dedicated small caps font for programs that don't correctly implement Opentype features. I had a little trouble with the current version using Python 3, but running the previous 2015 version in Python 2 worked perfectly. 

@jrmiller82 I'm pretty green when it comes to Python, but the process was pretty straightforward. The process is to 

1. Download [fonttools](https://github.com/fonttools/fonttools) and the [pyftfeatfreeze script](https://github.com/twardoch/fonttools-utils/tree/master/pyftfeatfreeze)
2. Install Python 2.7 (or use another tool to convert the old version to Python 3)
3. Navigate to where you downloaded the fonttools library, and execute "setup.py install" to install the library.
4. Go to where you kept the script and the font file and then follow the directions in the readme. For example, to convert SourceSerifPro-Regular.otf to a dedicated small caps font, use the following command. 
`pyftfeatfreeze.py -f smcp -S -U Caps -R "Source Serif Pro/Smart Serif Pro,SourceSerifPro/SmartSerifPro" SourceSerifPro-Regular.otf SourceSerifPro-Regular-Caps.otf`
This will add the Suffix "Caps" to the name, replace "Source Serif Pro" with "Smart Serif Pro," and replace "SourceSerifPro" with "SmartSerifPro." Please note, this is just an example of how the tool works, and I'm not an expert on font licenses or trademark law, so please make sure this usage does not run afoul of any of the license terms or trademarks.
5. Install the fonts on your system and use as needed in Word, Acrobat, etc.

I hope this helps! And thank you to @frankrolf for your work. It is simply beautiful, and I would love to see more of it in the legal world.
